executor/join_result_generators.go

```diff
@@ -32,21 +32,51 @@ var (
 	_ joinResultGenerator = &innerJoinResultGenerator{}
 )
 
-// joinResultGenerator is used to generate join results according the join type, see every implementor for detailed information.
+// joinResultGenerator is used to generate join results according the join type.
```

Comment:
joinResultGenerator is used to generate join results according **to** the join type.